library/std/src/sys/pal/unix/fs.rs

```diff
@@ -1327,7 +1333,7 @@ impl File {
             None => Ok(libc::timespec { tv_sec: 0, tv_nsec: libc::UTIME_OMIT as _ }),
         };
         cfg_if::cfg_if! {
-            if #[cfg(any(target_os = "redox", target_os = "espidf", target_os = "horizon"))] {
+            if #[cfg(any(target_os = "redox", target_os = "espidf", target_os = "horizon", target_os = "vxworks"))] {
                 // Redox doesn't appear to support `UTIME_OMIT`.
                 // ESP-IDF and HorizonOS do not support `futimens` at all and the behavior for those OS is therefore
                 // the same as for Redox.
```

Comment:
Insert VxWorks into the comment here. It just has to say whether it has UTIME_OMIT or futimens or whatever other reason it uses this support.

Reply:
Futimens and utime_omit may be available for vxworks. But they are not yet tested and unintroduced in libc crate. Can I use this temporary fix to get a successful build?

Reply:
I'll add this "`futimens` and `UTIME_OMIT` are a work in progress for vxworks. " in the comments.

Reply:
Yes, that seems fine.